### ISSUE TITLE
Downloadable metrics graphs

### ISSUE THREAD

**Issue by A-Artemis** [open-edge-platform/training_extensions#5656]
## Objective

Allow the user to download the training metrics

## Motivation

The user can view training-time metrics such as the loss over time, learning rate schedule, etc... in the UI, but he cannot download the raw data. The option to download the metrics data in a human-readable format can be helpful to users who need it for papers or presentations.

## Requirements

- Allow exporting the training-metrics in a human-readable format (e.g. image or .pdf) from the UI
